docs/resource-duration.md

```diff
@@ -9,7 +9,7 @@ information. This is intended to be an **indicative but not accurate** value.
 
 ## Configuration
 
-This is only set if [the feature flags is enabled in config](workflow-controller-configmap.yaml).
+This is turned on only if [the `resourcesDuration` flag is enabled in config](workflow-controller-configmap.yaml).
```

Comment:
I don't see the `resourcesDuration` flag anywhere in the config? Did I miss it?

Also, should it be `resourceDuration` (singular)?

Reply:
it is meant to be plural - but this config item is gone now - I'll correct